src/UsgsAstroSarSensorModel.cpp

```diff
@@ -427,9 +427,64 @@ csm::ImageCoordCovar UsgsAstroSarSensorModel::groundToImage(
     double* achievedPrecision,
     csm::WarningList* warnings) const
 {
-  return csm::ImageCoordCovar(0.0, 0.0,
-                              1.0, 0.0,
-                                   1.0);
+  // Ground to image with error propagation
+  // Compute corresponding image point
+  csm::EcefCoord gp;
```

Comment:
Does `gp(groundPt)` not do the right thing? Or `gp(groundPt.x, groundPt.y, groundPt.z)`. I like these two better than separating out into the next few lines.

Reply:
Yeah, we should actually be able to just pass the EcefCoordCovar into the functions and it will get sliced but that's okay. I think the whole making a copy part can just go away.

Reply:
so, we do have to make a new ground point for const reasons, but your suggestion of using the copy constructor works.